pkg/distro/defs/loader.go

```diff
@@ -214,6 +214,22 @@ func NewDistroYAML(nameVer string) (*DistroYAML, error) {
 			v := toplevel.ImageTypes[name]
 			v.name = name
 			foundDistro.imageTypes[name] = v
+			for idx := range v.Platforms {
+				templ, err := template.New("uefi-vendor").Parse(v.Platforms[idx].UEFIVendor)
+				if err != nil {
+					return nil, err
+				}
+				var buf bytes.Buffer
```

Comment:
I think this should also check for an empty vendor string, if that were to happen the UEFI files would end up in the wrong place in the EFI directory.